crates/prune/prune/src/segments/user/transaction_lookup.rs

```diff
@@ -122,15 +122,19 @@ mod tests {
         PruneCheckpoint, PruneInterruptReason, PruneLimiter, PruneMode, PruneProgress, PruneSegment,
     };
     use reth_stages::test_utils::{StorageKind, TestStageDB};
-    use reth_testing_utils::{generators, generators::random_block_range};
+    use reth_testing_utils::generators::{self, random_block_range, BlockParams};
     use std::ops::Sub;
 
     #[test]
     fn prune() {
         let db = TestStageDB::default();
         let mut rng = generators::rng();
 
-        let blocks = random_block_range(&mut rng, 1..=10, B256::ZERO, 2..3, None, None);
+        let blocks = random_block_range(
+            &mut rng,
+            1..=10,
+            BlockParams { parent: Some(B256::ZERO), tx_count: Some(0..2), ..Default::default() },
```

Comment:
hmm why is this change from `2..3` to `0..2` required?

Reply:
This is incorrect, it shouldn't start with `0` but instead `1..3`, but i think the test is rightfully failing